addons/web/static/lib/owl/owl.js

```diff
@@ -1641,7 +1641,7 @@
         return fiber;
     }
     function throwOnRender() {
-        throw new OwlError("Attempted to render cancelled fiber");
+        throw new OwlError("Attempted to render canceled fiber");

```

Comment:
Aren't you supposed to make a PR to the [Owl repository](https://github.com/odoo/owl)?